src/components/PathwayCourse/styles.js

```diff
@@ -72,9 +72,11 @@ const useStyles = makeStyles((theme) => ({
     paddingTop: "16px",
   },
   certificateLogo: {
-    // marginBottom : "5px",
     fontSize: "90px",
   },
+  link: {
+    color: "green",
```

Comment:
Use theme colors for consistency in styles and to allow for future changes to be made in one place.